web/src/main/java/com/futurewei/alcor/web/restclient/NodeManagerRestClient.java

```diff
@@ -30,4 +37,13 @@ public NodeInfoJson getNodeInfo(String nodeId) throws Exception {
         String url = nodeManagerUrl + "/" + nodeId;
         return getRequest(url, NodeInfoJson.class);
     }
+
+    @DurationStatistics
+    public List<NodeInfo> getNodeInfoByNodeName(String nodeName) throws Exception {
```

Comment:
@chenpiaoping Would like to keep you updated that this is the new contract between port manager and node manager.